pintc/src/asm_gen/tests.rs

```diff
@@ -513,16 +513,6 @@ fn binary_ops() {
                   Access(PredicateData)
                   Pred(Gt)
                 node 2
-                  Stack(Push(0))
```

Comment:
This unit tests seems to have
```
                constraint x > y;
                constraint x > y;
```
which is unnecessary. Let's remove one of them.